v22.1/ui-distributed-dashboard.md

```diff
@@ -0,0 +1,127 @@
+---
+title: Distributed Dashboard
+summary: The Distributed dashboard lets you monitor important distribution layer health and performance metrics.
+toc: true
+docs_area: reference.db_console
+---
+
+The **Distributed** dashboard lets you monitor important distribution layer health and performance metrics.
+
+To view this dashboard, [access the DB Console](ui-overview.html#db-console-access) and click **Metrics** on the left-hand navigation, and then select **Dashboard** > **Distributed**.
+
+{% include {{ page.version.version }}/ui/ui-metrics-navigation.md %}
+
+{{site.data.alerts.callout_info}}
+All timestamps in the DB Console are shown in Coordinated Universal Time (UTC).
+{{site.data.alerts.end}}
+
+The **Distributed** dashboard displays the following time series graphs:
+
+## Batches
+
+<img src="{{ 'images/v22.1/ui_batches.png' | relative_url }}" alt="DB Console batches graph" style="border:1px solid #eee;max-width:100%" />
+
+The **Batches** graph shows you various details about [`BatchRequest`](architecture/distribution-layer.html#batchrequest) traffic in the [Distribution layer](architecture/distribution-layer.html).html).
```

Comment:
Looks like a copy paste/formatting error. Not 100% this fixes it until after the build runs.
```suggestion
The **Batches** graph shows you various details about [`BatchRequest`](architecture/distribution-layer.html#batchrequest) traffic in the [Distribution layer](architecture/distribution-layer.html).
```